bot/client.py

```diff
@@ -252,7 +302,11 @@ async def loop(self, context: commands.Context):
         context.voice_state.loop = not context.voice_state.loop
         await context.message.add_reaction('✅')
 
-    @commands.command(aliases=['p'])
+    @commands.hybrid_command(
+        name='play',
```

Comment:
can you add `mi` as an alias? play can be used by a lot of bots

Reply:
Added `mi` as the default command and others as aliases.